utils/util.go

```diff
@@ -45,7 +45,7 @@ func Context() context.Context {
 
 var letters = []rune("abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ1234567890")
 
-func randSeq(n int) string {
+func RandSeq(n int) string {
```

Comment:
Made this method exportable, because 50 char token is required for viewing response

Reply:
Awesome! I'd forgotten this already existed.